### PR TITLE
Allow edit of priority field on EditCommand

### LINKED ISSUE #46: Support editing of remark and priority field of Person class
Tasks:
- [X] Implement edit of remark field on Person class (see PR #47)
- [X] Implement edit of priority field on Person class (see PR #66)

### PR DESCRIPTION
**Pull request by LeongWZ**
Resolves #46. 

This commit allows user to edit the priority field on `Person` class.
The user may edit the priority field by typing `edit INDEX pri/ priority_value"`, e.g. `edit 1 pri/ HIGH`.

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 1]
```diff
@@ -71,43 +75,20 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
...
+        helper.execute_specifiedRemarkField_shouldModifyRemark("This is a remark!");
     }
 
     @Test
     public void execute_specifiedEmptyRemarkField_shouldModifyRemark() {
-        execute_specifiedRemarkField_shouldModifyRemark("");
+        helper.execute_specifiedRemarkField_shouldModifyRemark("");
+    }
+
+    @Test void execute_specifiedPriorityField_shouldModifyPriority() {
```
Comment (Gra7ityIC3) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java]: Could you specify the access modifier and place the `@Test` annotation above the method signature?

**Review by Gra7ityIC3 — COMMENTED** [review]
Aside from some minor changes, LGTM.

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java (hunk 1 above)
Comment (LeongWZ) [reply src/test/java/seedu/address/logic/commands/EditCommandTest.java]: Noted. Thanks for catching that.

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 2]
```diff
@@ -255,13 +254,22 @@ public void execute_specifiedPriorityField_shouldModifyPriority(Priority priorit
...
 
-            String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS,
-                    Messages.format(personToBeExpected));
+            String expectedMessage = generateExpectedSuccessMessage(personToBeExpected);
 
             assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
         }
     }
+
+    private Model generateExpectedModel(Person personToBeEdited, Person personToBeExpected) {
```
Comment (TerenceWongsExtravaganza) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java:265]: Good job abstracting out the model creation process out of testcase!

**Review by Gra7ityIC3 — APPROVED** [review]
LGTM!

**Pull request merged**